cypress-dotnet-sdk/API References/cypress/includes/product.json

```diff
@@ -34,6 +34,12 @@
 		"duplicate_records": {
 			"type": "boolean"
 		},
+        "shift_records": {
+            "type": "boolean"
+        },
+        "measure_selection": {
```

Comment:
I think this is the wrong field.  should probably be "cert_edition"

measure_selection appears to indicate how the measures were selected (e.g., used the custom method) this is of little interest in the api.